crates/types/src/vote.rs

```diff
@@ -85,9 +85,12 @@ pub struct VoteAccumulator<
     /// Map of all signatures accumlated so far
     pub vote_outcomes: VoteMap2<Commitment<VOTE::Commitment>>,
     /// A list of valid signatures for certificate aggregation
-    pub sig_lists: Vec<<TYPES::SignatureKey as SignatureKey>::PureAssembledSignatureType>,
+    pub sig_lists: HashMap<
+        Commitment<VOTE::Commitment>,
+        Vec<<TYPES::SignatureKey as SignatureKey>::PureAssembledSignatureType>,
+    >,
     /// A bitvec to indicate which node is active and send out a valid signature for certificate aggregation, this automatically do uniqueness check
-    pub signers: BitVec,
+    pub signers: HashMap<Commitment<VOTE::Commitment>, BitVec>,
```

Comment:
Does it make sense to combine these all into one map?

Reply:
I was debating this I think it makes sense